R/bambu-extendAnnotations-utilityCombine.R

```diff
@@ -19,14 +19,15 @@ isore.combineTranscriptCandidates <- function(readClassList,
         min.readCount, min.readFractionByGene, 
         min.txScore.multiExon, min.txScore.singleExon, verbose) %>% data.table()
     combinedSplicedTranscripts[,confidenceType := "highConfidenceJunctionReads"]
-    combinedUnsplicedTranscripts <- 
-        combineUnsplicedTranscriptModels(readClassList, bpParameters, 
-        stranded, min.readCount, min.readFractionByGene, 
-        min.txScore.multiExon, min.txScore.singleExon, verbose) %>% data.table()
-    combinedUnsplicedTranscripts[, confidenceType := "unsplicedNew"]
-    combinedTranscripts <- as_tibble(rbindlist(list(combinedSplicedTranscripts,
-        combinedUnsplicedTranscripts), fill = TRUE))
-    return(combinedTranscripts)
+    return(combinedSplicedTranscripts)
```

Comment:
Hi @andredsim 

where is the commented code moved to? does this mean there is no unsplicedNew read classes?

Reply:
This is an outdated version, just double check you have pulled the latest version of this branch, as the commented code is uncommented in it